### PR TITLE
Add `Track` field to Session form

### PR DESCRIPTION
**Pull request by shivamMg**
- Remove `Sessions` field from track form
- Add `Track` field to session form
- Add datagetter for tracks
- Update track and session creation/modification functions

[Related Discussion](https://github.com/fossasia/open-event-orga-server/issues/274#issuecomment-217872505)

I think coveralls has not been configured. It is checking files from installed libraries: https://coveralls.io/builds/6104621

### CODE REVIEW

**Comment by leto** [comment]
Some tests would be nice 😄

File: tests/test_datagetter.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    def _delete_track(self):
+        DataManager().remove_track(1)
+
+    def test_track_field_at_session_form(self):
+        register(self.app, 'test_user', 'email@example.com', 'test_pwd')
+        # User is registered and logged in
+
+        with app.test_request_context():
+            self._create_track()
+            rv = self.app.get('/admin/event/1/session/new',
```
Comment (juslee) [inline tests/test_datagetter.py]: Codacy Issue found: [Invalid variable name "rv"](https://www.codacy.com/app/triplez/open-event-orga-server/file/2539694755/issues/source?bid=3233864&fileBranchId=3285268#l37)

File: tests/test_datagetter.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        form = TrackForm()
+        form.name.data = 'SomeTrack'
+        form.description.data = 'With Description'
+        form.track_image_url.data = 'https://example.com/image.gif'
+        DataManager().create_track(form, 1)
+
+    def _delete_track(self):
+        DataManager().remove_track(1)
+
+    def test_track_field_at_session_form(self):
```
Comment (juslee) [inline tests/test_datagetter.py]: Codacy Issue found: [Invalid method name "test_track_field_at_session_form"](https://www.codacy.com/app/triplez/open-event-orga-server/file/2539694755/issues/source?bid=3233864&fileBranchId=3285268#l31)

File: tests/test_datagetter.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+class TestSessionForm(OpenEventTestCase):
+
+    def setUp(self):
+        self.app = Setup.create_app()
+        with app.test_request_context():
+            db.session.add(ObjectMother.get_event())
+            db.session.commit()
+
+    def _create_track(self):
```
Comment (juslee) [inline tests/test_datagetter.py]: Codacy Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2539694755/issues/source?bid=3233864&fileBranchId=3285268#l21)

File: tests/test_datagetter.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        form = TrackForm()
+        form.name.data = 'SomeTrack'
+        form.description.data = 'With Description'
+        form.track_image_url.data = 'https://example.com/image.gif'
+        DataManager().create_track(form, 1)
+
+    def _delete_track(self):
```
Comment (juslee) [inline tests/test_datagetter.py]: Codacy Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2539694755/issues/source?bid=3233864&fileBranchId=3285268#l28)

**Comment by shivamMg** [comment]
@leto I've added test that checks session form for added track. :smile:

**Pull request merged**